spec/controllers/account_reset/confirm_request_controller_spec.rb

```diff
@@ -9,5 +9,14 @@
         expect(response).to redirect_to(root_url)
       end
     end
+
+    context 'email is present in flash' do
```

Comment:
Is this intentional? It doesn't seem directly related to the policy, but I might be missing a step. Otherwise, this looks reasonable to me.

Reply:
Yes, the controller has different logic depending on whether or not `flash[:email]` is present. We were only testing the absence not the presence. In the presence scenario, the controller makes a call to `SmsLoginOptionPolicy.new(current_user).configured?`